server/views/components/figure/index.njk

```diff
@@ -1,4 +1,4 @@
-<figure class="figure figure--{{type}}">
+<figure class="figure figure--{{type}} {% if full %}figure--full{% endif %}">
   {% if type === 'picture' %}
     <picture class="figure__picture">
```

Comment:
Just looking at it could we not put the `--full` modifier on the `picture`?
e.g.
```SASS
.figure__picture--full {
  // bleed
}
```

And then the caption always get's contained.